src/codegeneration/CommonJsModuleTransformer.js

```diff
@@ -59,106 +45,103 @@ export class CommonJsModuleTransformer extends ModuleTransformer {
     return tree.moduleName;
   }
 
-  moduleProlog() {
-    let statements = super.moduleProlog();
-
-    // declare temp vars in prolog
-    if (this.moduleVars_.length) {
-      let tmpVarDeclarations = createVariableStatement(createVariableDeclarationList(VAR,
-          this.moduleVars_.map((varName) => createVariableDeclaration(varName, null))));
-
-      statements.push(tmpVarDeclarations);
+  wrapModule(statements) {
+    if (this.needsInteropRequire_) {
+      const req = parseStatement `function $__interopRequire(id) {
+        id = require(id);
+        return id && id.__esModule && id || {default: id};
```

Comment:
I think  `id && id.__esModule ? id : {default: id}` makes the role of the second `id` clearer.